### PR TITLE
Fix unexpected whitespace of the command catalogs in several Redis plugins.

### PR DESCRIPTION
**Pull request by SWHHEART**
### Fix <bug description or the bug issue number or bug issue link>
- [ ] Add a unit test to verify that the fix works.
- [ ] Explain briefly why the bug exists and how to fix it.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Could you follow the template of the pull request template to change the `CHANGES`?

**Comment by SWHHEART** [comment]
> Could you follow the template of the pull request to change the `CHANGES`?
Sorry, I haven't studied this too much. Next time, you can inform your developer to fix this error

**Comment by wu-sheng** [comment]
> > Could you follow the template of the pull request to change the `CHANGES`?
> 
> Sorry, I haven't studied this too much. Next time, you can inform your developer to fix this error

What? I just need you to add the changes to the change logs.
I think you misunderstand my point.

File: CHANGES.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Release Notes.
 * Fix tomcat-10x-plugin and add test case to support tomcat7.x-8.x-9.x.
 * Fix thrift plugin generate duplicate traceid when `sendBase` error occurs
 * Support keep trace profiling when cross-thread.
+* Fix unexpected whitespace of the command catalogs in several Redis plugins.
```
Comment (wu-sheng) [inline CHANGES.md:24]: @SWHHEART I mean you missed this.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**